File: PTX3_LAZUR_MARCIN/code.rb
class SimpleClass
    def function(x=3, a=6, b=9)
        for i in 0..10
            puts i+i
            for k in 0..2
                puts i+k
                end
            end
        if x == 1
            return a
            end
        return b
        end
    end
end
 
obj = SimpleClass()
kek = obj.function(x=5)
puts kek

File: PTX3_LAZUR_MARCIN/code.py
class SimpleClass:
    def function(self, x=3, a=6, b=9):
        for i in range(10):
            print i+i
            for k in range(2):
                print i+k
        if x == 1:
            return a
        return b
 
obj = SimpleClass()
kek = obj.function(x=5)
print kek
